### ISSUE TITLE
Compatibility with Connect 3.x

### ISSUE THREAD

**Issue by mikehostetler** [tdebarochez/connect-couchdb#29]
Connect 3.0.0 breaks with the following error when trying to add the design docs:

``` sh
> $ node tools/put_design_docs.js _sessions 1500 admin admin                         ⬡ 0.10.28 [±master ●]

/Users/mikehostetler/Source/connect-couchdb/lib/connect-couchdb.js:4
    var Store = connect.session.Store;
                               ^
TypeError: Cannot read property 'Store' of undefined
    at module.exports (/Users/mikehostetler/Source/connect-couchdb/lib/connect-couchdb.js:4:32)
    at Object.<anonymous> (/Users/mikehostetler/Source/connect-couchdb/tools/put_design_docs.js:16:77)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

**Comment by paintedbicycle** [comment]
Same issue.

Did you find out how to use this with express instead of connect?

**Comment by tdebarochez** [comment]
Express is on top of connect : you cannot use express without it.

EDIT : Oops, I spoke too soon... Yes, it's the solution : I'm working on a fix.

**Comment by tdebarochez** [comment]
Fixed in #2ac1929

**Comment by paintedbicycle** [comment]
Tested locally and in production. Confirmed fixed. Thanks.
